File: server-support_study-develop/src/app/models/GroupCourse.js
const mongoose = require("mongoose")
const Schema = mongoose.Schema
const slug = require("mongoose-slug-generator")
mongoose.plugin(slug)
const Participant=require('./Participant')

const GroupCourse = new Schema(
  {
    createBy: { type: String, required: true },
    groupName: { type: String, required: true },
    groupDescription: { type: String },
    groupImage: { type: String },
    courseId: {
        type: mongoose.Schema.Types.ObjectId,
        required: true,
        ref: "Course",
      },
      participant: [{type: mongoose.Schema.Types.ObjectId,ref:"JoinGroup"}]

  },
  {
    timestamps: true,
  }
)
module.exports = mongoose.model("GroupCourse", GroupCourse)

File: SuportStudy-App/app/src/main/java/com/example/suportstudy/model/GroupCourse.kt
package com.example.suportstudy.model

data  class GroupCourse (
    var _id:String?=null,
    var createBy:String?=null,
    var groupName:String?=null,
    var groupDescription:String?=null,
    var groupImage:String?=null,
    var courseId:String?=null,
    var participant:List<Participant>?=null,
        )